snaps/features/transaction-insights.md

```diff
@@ -0,0 +1,112 @@
+---
+description: Provide transaction insights in MetaMask's transaction confirmation window.
+toc_max_heading_level: 4
+sidebar_position: 13
+---
+
+# Transaction insights
+
+You can provide transaction insights in MetaMask's transaction confirmation window before a user
```

Comment:
Perhaps add a high level overview about what transactions insights are, and why you may want to create one? Are there any limitations?